player_android.go

```diff
@@ -279,6 +292,19 @@ func (p *player) Write(data []uint8) (int, error) {
 }
 
 func (p *player) Close() error {
+	p.mu.Lock()
+	p.mu.Unlock()
```

Comment:
It seems IllegalStateException is throwed if release method is called for the AudioTrack instance that is already released.
I inserted these mutex lock/unlock and check nil of `p.audioTrack` to care for concurrent call of `Close()` function.